admin_scripts/ubuntu/protect_settings.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+set -ex
+
+if [ !  -f "/usr/bin/gnome-control-center.real" ]
+then
+    dpkg-divert --divert  /usr/bin/gnome-control-center --add /usr/bin/gnome-control-center.real
```

Comment:
I think this should be the other way round: `--divert /usr/bin/gnome-control-center.real --add /usr/bin/gnome-control-center`. (You probably also want the `--rename` option so that the file actually gets moved.)

Reply:
Yo're right, done!